### PR TITLE
Add run_id

### PR DESCRIPTION
**Pull request by joakim-hove**
**Task**
Followup to: https://github.com/Statoil/libres/pull/42

**Approach**
Changed function signatures.

**Pre un-WIP checklist**
- [ ] Statoil tests pass locally
- [x] Have completed graphical integration test steps

**Depends on**
* Statoil/libres#42

### CODE REVIEW

File: libenkf/applications/ert_tui/enkf_tui_analysis.c [hunk 1]
```diff
@@ -72,7 +72,8 @@ void enkf_tui_analysis_select_module__(void * arg) {
   menu_type * menu = arg_pack_iget_ptr( arg_pack , 1 );
   
   {
-    analysis_config_type * analysis_config = enkf_main_get_analysis_config( enkf_main );
+    // This is really a const return.
+    analysis_config_type * analysis_config = (analysis_config_type *) enkf_main_get_analysis_config( enkf_main );
```
Comment (markusdregi) [inline libenkf/applications/ert_tui/enkf_tui_analysis.c]: I find this to be a bad idea! We should not silence this warning. I would prefer that we make `enkf_main_get_analysis_config` not return a `const` until `analysis_config` some time in the future actually is a const (if this ever happens).

File: libenkf/applications/ert_tui/enkf_tui_analysis.c [hunk 2]
```diff
@@ -98,7 +99,7 @@ void enkf_tui_analysis_list_modules__(void * arg) {
 
 void enkf_tui_analysis_reload_module__(void * arg) {
   enkf_main_type * enkf_main = enkf_main_safe_cast( arg );
-  analysis_config_type * analysis_config = enkf_main_get_analysis_config( enkf_main );
+  analysis_config_type * analysis_config = (analysis_config_type *) enkf_main_get_analysis_config( enkf_main );
```
Comment (markusdregi) [inline libenkf/applications/ert_tui/enkf_tui_analysis.c]: Same here..

File: python/python/ert_gui/simulation/models/ensemble_experiment.py [hunk 3]
```diff
@@ -1,37 +1,53 @@
...
         self.setPhaseName("Pre processing...", indeterminate=True)
-        self.ert().getEnkfSimulationRunner().createRunPath(active_realization_mask, 0)
+        self.ert().getEnkfSimulationRunner().createRunPath( run_context )
         self.ert().getEnkfSimulationRunner().runWorkflows( HookRuntime.PRE_SIMULATION )
 
         self.setPhaseName( run_msg, indeterminate=False)
-
-        num_successful_realizations = self.ert().getEnkfSimulationRunner().runEnsembleExperiment(job_queue, active_realization_mask)
-        self.assertHaveSufficientRealizations(num_successful_realizations, active_realizations)
+
```
Comment (markusdregi) [inline python/python/ert_gui/simulation/models/ensemble_experiment.py]: Please remove

**Comment by joakim-hove** [comment]
OK - I have removed the const hackery; unfortunately the libres is back to red status ....

**Pull request merged**